values.html

```diff
@@ -97,16 +99,16 @@ <h2 class="dwyl-mint bb dwyl-b--gray pl3">dwyl</h2>
 
     <section class="dwyl-bg-dark-gray white pt5 pb4 pb5-ns ph4 ph5-m ph5-l center">
       <div class="mw8 center">
-        <h2 class="f2 mv0">More Than Just Software</h2>
+        <h2 class="f2 mv0">More than just software</h2>
         <p class="istok-web">We solve real problems with technology. We are different by design.</p>
         <a href="/community" class="white link">
-          <button class="bn dib link mb3 ph3 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4 pointer istok-web">Join Our Community</button>
+          <button class="bn dib link mb3 mr2 ph3 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4 pointer istok-web">Join Our Community</button>
         </a>
         <a href="/team" class="white link">
-          <button class="bn dib link ml2 mb3 ph3 pv2 dwyl-bg-mint white br2 f5 shadow-4 pointer istok-web">Meet the Team</button>
+          <button class="bn dib link mb3 ph3 pv2 dwyl-bg-mint white br2 f5 shadow-4 pointer istok-web">Meet the Team</button>
```

Comment:
There seems to be less padding between the 'More than Just Software' section and the 'We believe in' section than between the value themselves:
<img width="978" alt="screen shot 2017-10-09 at 19 40 13" src="https://user-images.githubusercontent.com/4185328/31353286-de79f8bc-ad29-11e7-8032-f7f1c1a1e165.png">

This was quite confusing to the eye when I first scrolled down as I wasn't sure where to focus and this looked like one big glob of text.